x/execution/client/rest/query.go

```diff
@@ -132,6 +132,9 @@ func queryListHandlerFn(cliCtx context.CLIContext) http.HandlerFunc {
 		sort.Sort(sort.Reverse(execution.ByBlockHeight(execs)))
 
 		start, end := client.Paginate(len(execs), page, limit, limit)
+		w.Header().Set("X-Page", fmt.Sprintf("%d", page))
+		w.Header().Set("X-Limit", fmt.Sprintf("%d", limit))
+		w.Header().Set("X-Total-Count", fmt.Sprintf("%d", len(execs)))
```

Comment:
don't use `fmt.Sprintf` for type conversion:
```suggestion
		w.Header().Set("X-Page", strconv.Itoa(page))
		w.Header().Set("X-Limit", strconv.Itoa(limit))
		w.Header().Set("X-Total-Count", strconv.Itoa(len(execs)))
```

Reply:
don't forget to import `strconv`